sdk/keyvault/azure-security-keyvault-certificates/test/samples/certificate-basic-operations/certificate_basic_operations.cpp

```diff
@@ -23,19 +20,21 @@
 #include <iostream>
 #include <thread>
 
+#include "get_env.hpp"
```

Comment:
The ultimate goal for the any sample we have should be that I, as a customer, can copy any folder whitin the test/samples/ directory to some other directory ( like ~/myApp ). Then I would expect I can cd into it and build it by running `cmake ..`

That means that the sample should manage on how to get its dependencies.

The current approach makes this difficult as the get-env library is expected to be included as a cmake subdirectory before the sample.

Following that we do for gtest would be better for this reason as well

Reply:
Yes, if we leave it like this, the fetchcontent PR will have to fetch this one as well, like it does others.